Sources/Caching/DeviceCache.swift

```diff
@@ -160,10 +160,14 @@ class DeviceCache {
 
     // MARK: - Offerings
 
-    func cachedOfferingsContentsData(appUserID: String) -> Data? {
-        return self.userDefaults.read {
-            $0.data(forKey: CacheKey.offerings(appUserID))
+    func cachedOfferingsContents(appUserID: String) -> Offerings.Contents? {
+        // Large data used to be stored in the user defaults and resulted in crashes, we need to ensure that
+        // we are cleaning out that data
+        userDefaults.write { defaults in
+            defaults.removeObject(forKey: CacheKey.offerings(appUserID))
         }
+
+        return largeItemCache.value(forKey: CacheKey.offerings(appUserID))
```

Comment:
Wouldn't this be the same as this?

```suggestion
        return self.value(for: CacheKey.offerings(appUserID))
```
To avoid repeating the exact same code